public/player.js

```diff
@@ -35,32 +35,42 @@
     YT.ready(onYouTubeIframeAPIReady);
   };
 
-  function toggleHighlight(x) {
-    x.classList.toggle("highlighted-option")
-    setTimeout(function() {
-      x.classList.remove("highlighted-option")
-    }, 3000)
+  var funcCount = (function () {
+      var counter = 0;
+      return function () {return counter += 1;}
+  })();
+
+  function toggleHighlight(answer, arr, event) {
```

Comment:
@des-des `arr` is actually an array. In my `onPlayerStateFunction` I converted the NodeList into an array: 
` var answersArr = Array.from(document.getElementsByTagName("FIGURE"));`

However, you are right, `arr` is not a great name. I will give it another name.